homepage/theme/templates/index.html

```diff
@@ -23,131 +23,501 @@ <h1 class="f1-ns f2 normal lh-solid ma0 mb3">
+                Fair & Secure
+              </em>
+            </h2>
+            <p class="wx18 center ma0">
+              Pay respect <i>to your website visitors. Gain valuable insights at the same time. All data is</i> encrypted end-to-end.
+            </p>
+            </div>
+
           </div>
+        </div>
+
       </div>
+
+    </div>
   </div>
 
-  <div class="w-100 cclr-mid-black white">
-    <div class="pt3 pb6 ph4" id="bg-cta-top">
-      <div class="flex justify-end mw6 center pt5">
-        <div class="w55 tr">
-          <h2 class="f25 normal lh-solid ma0 mb3">
-            In the making
-          </h2>
-          <p class="ma0 mb4">
-            Offen is still in alpha. Discover what is already up and running and where we want to go in the coming months.
-          </p>
-          <a class="b link dim ph4 pv2 dib b--solid bw2 b--white white" href="/blog/">Project status</a>
-        </div>
+<!-- INSIGHTS -->
+  <div class="w-100 dib bg-white">
+    <div class="tc mh4">
+      <p class="f1-ns f2 lh-solid ma0 mt6">
+        Gain insights in an ethical way
+      </p>
+      <div class="mw7 center">
+        <p class="f5 ma0 mt3 mb6 mh0 mh3-m mh5-l">
+          Web analytics <i>is still an unfair and impenetrable business.</i> A proper alternative to common tools <i>is missing. Therefore we develop one.</i> <a class="b link dim cclr-fnt-black-mid" href="/deep-dive/">Deep dive</a>
+        </p>
       </div>
     </div>
   </div>
 
-  <div class="w-100 bg-white gray">
-    <div class="flex flex-wrap justify-center pt4 pb5">
-      <div class="mw7 mw8-l center ph5-ns ph3">
-        <h2 class="f25 normal lh-solid">Summary</h2>
-        <div class="columns mb3">
-          {{ page.content }}
-          <a class="b link dim ph4 pv2 mt3 dib b--solid bw2 brd-cclr-mid-yellow fnt-cclr-mid-yellow" href="/deep-dive/">Deep dive</a>
+  <!-- COMPARE -->
+  <div class="w-100" id="bg-explainer">
+    <div class="mw8 center flex flex-wrap justify-center">
+
+      <!-- COMPARE A -->
+      <div class="w-50-l w-100 cclr-bg-yellow-mid">
+
+        <div class="w-100 dib">
+          <div class="wx24 center">
+
+            <div class="relative">
+              <div class="absolute bg-compare-a-pfeil">
+                <img src="/theme/images/gfx-compare-a-pfeil.png" alt="arrow" width="40px" height=="120px">
+              </div>
+            </div>
+
+            <p class="f25 lh-solid ma0 mt6 mb3">
+              Your job
+            </p>
+          </div>
+        </div>
+
+        <div class="w-100 hx24 dib cclr-bg-white-fade">
+          <div class="wx24 center">
+
+            <div class="flex flex-wrap justify-center">
+              <div class="w-20 mt4">
+                <img src="/theme/images/gfx-compare-a.png" alt="numbers" width="75px" height="315px">
+              </div>
+              <div class="w-80 mt4">
+                <p class="ma0 ml3">
+                  Self host <i>Offen and thereby comply with GDPR guidelines.</i>
```

Comment:
"...in compliance with GDPR guidelines"?

Reply:
:+1: